tumblr-gif-search/app.py

```diff
@@ -24,51 +27,58 @@ def config():
     os.environ['JINA_PORT'] = os.environ.get('JINA_PORT', str(45678))
 
 
-# for index
-def index():
+def index(num_docs: int):
     f = Flow.load_config('flow-index.yml')
-
+    num_docs = min(num_docs, len(glob(GIF_BLOB)))
     with f:
-        f.index_files(GIF_BLOB, request_size=10, read_mode='rb', skip_dry_run=True)
+        with TimeContext(f'QPS: indexing {num_docs}', logger=f.logger):
+            f.index_files(GIF_BLOB, request_size=10, read_mode='rb', skip_dry_run=True, size=num_docs)
 
 
-# for search
-def search():
+def query_restful():
     f = Flow.load_config('flow-query.yml')
+    f.use_rest_gateway()
 
+    # no perf measure, as it opens a REST api and blocks
```

Comment:
same as above

Reply:
As mentioned above, here it may not be worth it to add the perf. measure